internal/util/quota/quota_constant.go

```diff
@@ -0,0 +1,106 @@
+			internalpb.RateScope_Cluster: {
+				internalpb.RateType_DDLCollection: &quotaConfig.DDLCollectionRate,
+				internalpb.RateType_DDLPartition:  &quotaConfig.DDLPartitionRate,
+				internalpb.RateType_DDLIndex:      &quotaConfig.MaxIndexRate,
+				internalpb.RateType_DDLFlush:      &quotaConfig.MaxFlushRate,
+				internalpb.RateType_DDLCompaction: &quotaConfig.MaxCompactionRate,
+				internalpb.RateType_DMLInsert:     &quotaConfig.DMLMaxInsertRate,
+				internalpb.RateType_DMLUpsert:     &quotaConfig.DMLMaxUpsertRate,
+				internalpb.RateType_DMLDelete:     &quotaConfig.DMLMaxDeleteRate,
+				internalpb.RateType_DMLBulkLoad:   &quotaConfig.DMLMaxBulkLoadRate,
+				internalpb.RateType_DQLSearch:     &quotaConfig.DQLMaxSearchRate,
+				internalpb.RateType_DQLQuery:      &quotaConfig.DQLMaxQueryRate,
+			},
+			internalpb.RateScope_Database: {
+				internalpb.RateType_DDLCollection: &quotaConfig.DDLCollectionRatePerDB,
+				internalpb.RateType_DDLPartition:  &quotaConfig.DDLPartitionRatePerDB,
+				internalpb.RateType_DDLIndex:      &quotaConfig.MaxIndexRatePerDB,
+				internalpb.RateType_DDLFlush:      &quotaConfig.MaxFlushRatePerDB,
+				internalpb.RateType_DDLCompaction: &quotaConfig.MaxCompactionRatePerDB,
+				internalpb.RateType_DMLInsert:     &quotaConfig.DMLMaxInsertRatePerDB,
+				internalpb.RateType_DMLUpsert:     &quotaConfig.DMLMaxUpsertRatePerDB,
+				internalpb.RateType_DMLDelete:     &quotaConfig.DMLMaxDeleteRatePerDB,
+				internalpb.RateType_DMLBulkLoad:   &quotaConfig.DMLMaxBulkLoadRatePerDB,
+				internalpb.RateType_DQLSearch:     &quotaConfig.DQLMaxSearchRatePerDB,
+				internalpb.RateType_DQLQuery:      &quotaConfig.DQLMaxQueryRatePerDB,
+			},
+			internalpb.RateScope_Collection: {
+				internalpb.RateType_DMLInsert:   &quotaConfig.DMLMaxInsertRatePerCollection,
+				internalpb.RateType_DMLUpsert:   &quotaConfig.DMLMaxUpsertRatePerCollection,
+				internalpb.RateType_DMLDelete:   &quotaConfig.DMLMaxDeleteRatePerCollection,
+				internalpb.RateType_DMLBulkLoad: &quotaConfig.DMLMaxBulkLoadRatePerCollection,
+				internalpb.RateType_DQLSearch:   &quotaConfig.DQLMaxSearchRatePerCollection,
+				internalpb.RateType_DQLQuery:    &quotaConfig.DQLMaxQueryRatePerCollection,
+				internalpb.RateType_DDLFlush:    &quotaConfig.MaxFlushRatePerCollection,
+			},
+			internalpb.RateScope_Partition: {
+				internalpb.RateType_DMLInsert:   &quotaConfig.DMLMaxInsertRatePerPartition,
+				internalpb.RateType_DMLUpsert:   &quotaConfig.DMLMaxUpsertRatePerPartition,
+				internalpb.RateType_DMLDelete:   &quotaConfig.DMLMaxDeleteRatePerPartition,
+				internalpb.RateType_DMLBulkLoad: &quotaConfig.DMLMaxBulkLoadRatePerPartition,
+				internalpb.RateType_DQLSearch:   &quotaConfig.DQLMaxSearchRatePerPartition,
+				internalpb.RateType_DQLQuery:    &quotaConfig.DQLMaxQueryRatePerPartition,
+			},
+		}
+	})
+}
+
+func GetQuotaConfigMap(scope internalpb.RateScope, params *paramtable.ComponentParam) map[internalpb.RateType]*paramtable.ParamItem {
```

Comment:
ensure params is a global object, or the second object will not be initialized. We'd better add the method comment.